hack/.ci/run-e2e-gke.sh

```diff
@@ -54,32 +54,64 @@ function kubectl_create {
 }
 
 function gather-artifacts {
-  kubectl -n=e2e run --restart=Never --image="${SO_IMAGE}" --labels='app=must-gather' --command=true must-gather -- bash -euExo pipefail -O inherit_errexit -c "function wait-for-artifacts { touch /tmp/done && until [[ -f '/tmp/exit' ]]; do sleep 1; done } && trap wait-for-artifacts EXIT && mkdir /tmp/artifacts && scylla-operator must-gather --all-resources --loglevel=2 --dest-dir=/tmp/artifacts"
+  kubectl_create -n=e2e -f=- <<EOF
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    labels:
+      app: must-gather
+    name: must-gather
+  spec:
+    restartPolicy: Never
+    containers:
+    - name: wait-for-artifacts
+      command:
+      - bash
+      - -euExo
+      - pipefail
+      - -O
+      - inherit_errexit
+      - -c
+      - sleep infinity
+      image: "${SO_IMAGE}"
+      imagePullPolicy: Always
+      volumeMounts:
+      - name: artifacts
+        mountPath: /tmp/artifacts
+    - name: must-gather
+      command:
+      - scylla-operator
+      args:
```

Comment:
optional nit: if you are changing command, I'd say you can specify the options there. separate args are mean for use with a "default" command)

Reply:
removed the command since it's the default